lib/ResponseDefinitions.php

```diff
@@ -123,6 +123,12 @@
  *     isThread?: bool,
  *     threadTitle?: string,
  *     threadReplies?: int,
+ *     metaData?: array{
+ *         pinnedByType?: string,
+ *         pinnedById?: string,
+ *         pinnedByName?: string,
```

Comment:
For editing, it's this:

```
 *     lastEditActorDisplayName?: string,
 *     lastEditActorId?: string,
 *     lastEditActorType?: string,
 ```

should we align the names? Then it would be `pinnedByActorType`, `pinnedByActorId`, ...

No hard opinion here, just noticed it.

Reply:
Yes, we can skip the by part